recws.go

```diff
@@ -404,25 +404,27 @@ func (rc *RecConn) connect() {
 		rc.httpResp = httpResp
 		rc.mu.Unlock()
 
+
 		if err == nil {
 			if !rc.getNonVerbose() {
 				log.Printf("Dial: connection was successfully established with %s\n", rc.url)
+			}
 
-				if !rc.hasSubscribeHandler() {
-					return
-				}
-
+			if rc.hasSubscribeHandler() {
 				if err := rc.SubscribeHandler(); err != nil {
-					log.Fatalf("Dial: connect handler failed with %s", err.Error())
+					if !rc.getNonVerbose() {
```

Comment:
here we should not hide the log.Fatal error message 
please just remove the getNonVerbose logic at this point

Reply:
True, I guess I was tired